code/src/components/TaskList.js

```diff
@@ -0,0 +1,48 @@
+import React from 'react'
+import { useSelector, useDispatch } from 'react-redux'
+import { TaskListStyling, TaskItem, DeleteButton, NumberOfTasks } from 'styles'
```

Comment:
Your approach here taught me a lot about styled components! Really cool how you used it through a separate styles file!